fastlane/Fastfile

```diff
@@ -337,6 +337,7 @@ SUPPORTED_LOCALES = [
       # Because the screenshot tests are together with other UI tests, we need
       # to specify to run only them and not all of the others.
       use_tests_in_classes: "com.automattic.simplenote.screenshots.ScreenshotTest",
+      use_timestamp_suffix: false,
```

Comment:
This way, when / if we'll run the composition, our code will be able to uniquely identify each screenshot.